front/lib/plans/usage/seats.ts

```diff
@@ -0,0 +1,37 @@
+import type { LightWorkspaceType } from "@dust-tt/types";
+import type Stripe from "stripe";
+
+import { Workspace } from "@app/lib/models";
+import { updateStripeQuantityForSubscriptionItem } from "@app/lib/plans/stripe";
+import { MembershipResource } from "@app/lib/resources/membership_resource";
+import { renderLightWorkspaceType } from "@app/lib/workspace";
+
+export async function countActiveSeatsInWorkspace(
+  workspaceId: string
+): Promise<number> {
+  const workspace = await Workspace.findOne({
+    where: {
+      sId: workspaceId,
+    },
+  });
+  if (!workspace) {
+    throw new Error(`Workspace not found for sId: ${workspaceId}`);
+  }
+
+  return MembershipResource.getMembersCountForWorkspace({
+    workspace: renderLightWorkspaceType({ workspace }),
+    activeOnly: true,
+  });
+}
+
+export async function reportSeatMembers(
```

Comment:
nit: would feel cleaner if functions were named `countActiveSeats`/`reportActiveSeats`

Reply:
Yep agree, I mainly copied/pasted the function from `workspace_usage` but will rename.